include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -203,15 +203,15 @@ def BoundedModelCheckingOp : VerifOp<"bmc", [
     match the types yielded by the `init` region).
   }];
 
-  let arguments = (ins I32Attr:$bound, I32Attr:$num_regs);
+  let arguments = (ins I32Attr:$bound, I32Attr:$num_regs, ArrayAttr:$initial_values);
```

Comment:
Maybe add some comment/TODO to indicate that this attribute is only a temporary solution until we support it in init regions?